static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -1002,10 +1002,6 @@
 
           const path = treeNodeTO.uri;
 
-          const loaderItems = CrafterCMSNext.util.content.getNumOfMenuOptionsForItem(
-            CrafterCMSNext.util.content.parseLegacyItemToDetailedItem(treeNodeTO)
-          );
```

Comment:
Considering legacy, perhaps we do need a loader on the Megamenu. Check how it looks with throttled network opening menu on legacy.